tgui/packages/tgui/interfaces/NtosPay.tsx

```diff
@@ -72,10 +72,10 @@ const Introduction = (props) => {
   return (
     <Section textAlign="center">
       <Table>
-        <Table.Row>Hi, {name}.</Table.Row>
-        <Table.Row>Your pay token is {owner_token}.</Table.Row>
+        <Table.Row>Привет, {name}.</Table.Row>
+        <Table.Row>Ваш платежный токен {owner_token}.</Table.Row>
         <Table.Row>
-          Account balance: {money} credit{money === 1 ? '' : 's'}
+          Баланс: {money} Кредит{money === 1 ? '' : 'ов'}
```

Comment:
```suggestion
          Баланс: {money} кр.
```
Русский сложнее в числах, чем добавление s. Вертимся-крутимся.

Reply:
жесть ты изворотливый